src/server/status/index.js

```diff
@@ -53,6 +56,9 @@ export default function (kbnServer, server, config) {
   server.route(wrapAuth({
     method: 'GET',
     path: '/status',
+    config: {
+      tags: ['api']
```

Comment:
confirmed the tag in /status after using `npm start -- --no-base-path`, added a new commit that reverts the tag